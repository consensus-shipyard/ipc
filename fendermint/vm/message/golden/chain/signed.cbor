a1665369676e6564828a1bffffffffffffffff5502ff6c5decc43f0657d653dfc888db04f96a0c39a75501c98d2f1c75ceb5912aff7f00b5a60032813a75ac1b114a444e81516e02584100bda3b9d4d5d7731b9cf74b18969735bf1133c8b7e5b792483199253ce8df09fcd989e63e36a4fa8865b53a4af1811f5a0dafc5eaa46d8e9a00b372a92989bf901be773d122bb50522a5841007082baf7d61bb3beb54f278f89485c04c21ab5ac8a02fd40294ed359a75dee43189e6e1f7b1076fee80135431910f79e56547cb850f34a7defed565e2a0620fe5100e2165f5473fcbb6c25b7c9f62f9e08a31bdca1f9db3b94acf8428701420104