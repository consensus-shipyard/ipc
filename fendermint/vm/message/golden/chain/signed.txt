Signed(SignedMessage { message: Message { version: 18446744073709551615, from: Address { payload: Secp256k1([201, 141, 47, 28, 117, 206, 181, 145, 42, 255, 127, 0, 181, 166, 0, 50, 129, 58, 117, 172]) }, to: Address { payload: Actor([255, 108, 93, 236, 196, 63, 6, 87, 214, 83, 223, 200, 136, 219, 4, 249, 106, 12, 57, 167]) }, sequence: 1245883350889098754, value: TokenAmount(9932229375525565691377146959193765155762615947792137979058505954726871205960593142920721455420987050970777344255324126610211082198096402.32226385693414184), method_num: 15898262879587314936, params: RawBytes { 8701 }, gas_limit: 16677903792184775210, gas_fee_cap: TokenAmount(5892661686900812402916430579778595043239110941386249057983897934879165380375437615565885206289661643046275694223485224682524740007133170.002394836476764414), gas_premium: TokenAmount(300521691098253062104.478788388166437027) }, signature: Signature { sig_type: Secp256k1, bytes: [4] } })